ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/lacp.yml

```diff
@@ -1,8 +1,8 @@
+---
 lacp:
   port_id:
     range:
       begin: 1
       end: 128
   rate_limit:
     default: false
-  system_priority: 0
```

Comment:
LACP system-priority configuration is not allowed if Mlag State is Active( primary/secondary )

Reply:
This should not stop us from testing it. The config is valid, but you are not allowed to _change it_ if MLAG is already up.

Reply:
Yes so MLAG config is also there in host1.cfg file so it give issue while apply though CVP hence i removed priority from here and added into host2